docs/digital-nomad.md

```diff
@@ -131,3 +131,5 @@ Resources for property insurance:
 - [11 Credit Card Insurance Benefits](https://www.creditcards.com/credit-card-news/card-travel-insurance-benefits-1273.php)
 
 - [The Traveler’s Guide to Personal Property Insurance](https://www.internationalinsurance.com/news/personal-property.php)
+
+![The New Digital Economy](./assets/divider-illustrations/divider-1.png)
```

Comment:
@lawlere FYI, the text in the `[ ]` section is the image alt text. You can't see it, but screenreaders and SEO can. So, we need to set the image alt text for each of these. I'll go through and revise this one, but please be careful in the future.